src/analyze.rs

```diff
@@ -101,7 +101,7 @@ crate fn find_suitable_param_names(trait_def: &ItemTrait) -> (Ident, Lifetime) {
 /// but this is cleaner and just the correct thing to do.
 #[cfg(feature = "nightly")]
 fn param_span() -> Span2 {
-    ::proc_macro::Span::def_site().into()
+    crate::proc_macro::Span::def_site().into()
```

Comment:
Ah, I think I've hit this same issue in recent 2018 previews with the `test` crate.